src/tools/clippy/clippy_lints/src/manual_async_fn.rs

```diff
@@ -105,7 +105,7 @@ fn future_trait_ref<'tcx>(
     cx: &LateContext<'tcx>,
     ty: &'tcx Ty<'tcx>,
 ) -> Option<(&'tcx TraitRef<'tcx>, Vec<LifetimeName>)> {
-    if let TyKind::OpaqueDef(item_id, bounds, false) = ty.kind
```

Comment:
This is the only change to semantics in clippy, but I don't think we need to explicitly opt out AFIT here.

Reply:
I don't think this is doing anything right now. We don't have tests for traits for this lint. I added one and changing `false`->`_` in the current implementation doesn't change the behavior there. So removing this should be fine :+1:

Reply:
Thank you for checking! ❤️